biojava-modfinder/pom.xml

```diff
@@ -54,12 +54,15 @@
 			<artifactId>log4j-core</artifactId>
 		</dependency>
 		<dependency>
-			<groupId>javax.xml.bind</groupId>
-			<artifactId>jaxb-api</artifactId>
+			<groupId>jakarta.xml.bind</groupId>
+			<artifactId>jakarta.xml.bind-api</artifactId>
+			<version>4.0.0</version>
```

Comment:
Please remove version here. It should be only in `dependencyManagement` in top pom.xml